src/components/SystemApp.tsx

```
@@ -0,0 +1,502 @@
+        </div>
+        <div className="flex items-center gap-3">
+          {lastUpdated && (
+            <span className="text-[10px]" style={{ color: "#a0a0b0" }}>
+              Updated {lastUpdated}
+            </span>
+          )}
+          <button
+            onClick={fetchStats}
+            className="p-1 rounded hover:bg-white/10 transition-colors"
+            title="Refresh now"
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)